Documentation/building/windows-instructions.md

```diff
@@ -8,44 +8,81 @@ MSBuild version 15 or later is required.
 
 # Building Xamarin.Android
 
- 1. Install the [build dependencies](dependencies.md).
+ 1. Run the [Visual Studio Installer](https://visualstudio.microsoft.com/vs/).
```

Comment:
For this first draft, I put all the instructions for setting up the build dependencies directly into this document, but I'm not sure if this is a better approach than adding a Windows section to the [Build Dependencies](https://github.com/xamarin/xamarin-android/blob/master/Documentation/building/dependencies.md) page. So I'd be happy to change that.

Reply:
I also made the instructions fairly specific to the current dependencies and versions. I'm happy to update them again when the first non-preview version of Visual Studio 2019 comes out, for example.